packages/core/useConfirmDialog/index.md

```diff
@@ -0,0 +1,238 @@
+---
+category: Utilities
+---
+
+# useConfirmDialog
+
+Creates event hooks to support modals and confirmation dialog chains. It returns 3 functions and 3 hooks.
+
+Functions can be used on the template, and hooks are a handy skeleton for the business logic of modals dialog or other actions that require user confirmation.
+
+## Functions and hooks
+
+- `reveal()` - triggers `onReveal` hook and sets `show.value` to `true`. Returns promise that resolves by `confirm()` or `cancel()`.
```

Comment:
sorry ^^' but just reading this `reveal()` now leads me to think that `prompt()` could be better 🤔 
But maybe we should ask some feedback from other parties 🤔 

_I'm even struggling with myself..._

Reply:
... on the other side ... we are free to rename it on our own 🤷 So it should not block this PR to get merged

Reply:
I agree with you but `prompt` is a built-in function in js. So this can make a mess.

Reply:
Maybe `evoke`?

Reply:
> ... is a built-in function in js

Oh 😲, nevermind, go with `reveal` then 👍

Reply:
`Confirm` is also a built-in function, but it doesn't make any problems.  Good names are always a problem. So `prompt` is next in line again.